<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>75</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Choose</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Option 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Option 2</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="ChangingWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
